src/lib/viewers/controls/icons/IconVolumeLow24.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+function IconVolumeLow24(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg focusable={false} height={24} viewBox="-3 5 24 24" width={24} {...props}>
```

Comment:
Are these negative values in the viewBox worrisome?

Reply:
The SVGs are copied directly from the existing icons. I'd rather not mess with them much.